README.md

```diff
@@ -1,23 +1,35 @@
-# Overwolf Boilerplate
+# Game Ticker
 
-> Quickly bootstrap a new project with Overwolf Boilerplate.
+**The social network for Gamers!**
 
-This boilerplate contains all the tools you need to build a modern web app with TypeScript, React, Vite, Storybook and Express.  
-You can use it to quickly bootstrap your project.
+Automatically share highlights and comments on your favorite games with your friends.
 
-ESLint, stylelint, prettier, husky and lintstaged are configured to give you a solid development experience.
+This project is the result of a [Gaming App Course](https://coda.io/@leon-machens/gaming-app-lvl).
```

Comment:
**The social network for Gamers**

Automatically share highlights and comments of your favorite games with your friends.

This project is the result of a [Gaming App Course](https://coda.io/@leon-machens/gaming-app-lvl) by Leon Machens.